go/runtime/bundle/manager.go

```diff
@@ -236,20 +247,44 @@ func (m *Manager) Queue(runtimeID common.Namespace, manifestHashes []hash.Hash)
 	}
 	m.downloadQueue[runtimeID] = hashes
 
-	// Trigger immediate download of new bundles.
+	// Trigger immediate download and clean-up of stale bundles.
 	select {
-	case m.downloadCh <- struct{}{}:
+	case m.triggerCh <- struct{}{}:
 	default:
 	}
 }
 
-// Download tries to download bundles in the queue.
-func (m *Manager) Download() {
+// Cleanup initiates the clean-up of regular bundles for the specified runtime,
+// removing versions lower than the specified one.
+func (m *Manager) Cleanup(runtimeID common.Namespace, version version.Version) {
+	m.mu.Lock()
+	defer m.mu.Unlock()
+
+	m.cleanupQueue[runtimeID] = version
+
+	// Trigger immediate download and clean-up of stale bundles.
+	select {
+	case m.triggerCh <- struct{}{}:
+	default:
+	}
+}
+
+// download tries to download bundles in the queue.
+func (m *Manager) download() {
 	for runtimeID := range m.runtimeIDs {
 		m.downloadBundles(runtimeID)
 	}
 }
 
+// clean removes outdated manifest and deletes corresponding
+// exploded bundles for runtimes in the clean-up queue.
+func (m *Manager) clean() {
+	m.logger.Info("cleaning regular bundles")
```

Comment:
```suggestion
	m.logger.Info("cleaning bundles")
```
I think I already suggested to add a similar log to `download`. Also, this log contradicts the function name, so what is correct: regular or stale? Confusing.

Reply:
I will add a log. Not sure why it hurts to be more specific?

Reply:
- Node operators will have no idea what regular bundle is, which can be confusing.
- Word regular gives you no extra information, so it is useless. 
- The log will be incorrect if we decide not to use the term regular bundle anymore.
- Larger logs.